objects/src/transaction/executed_tx.rs

```diff
@@ -100,7 +100,12 @@ impl ExecutedTransaction {
 
     /// Returns a reference to the transaction script.
     pub fn tx_script(&self) -> Option<&TransactionScript> {
-        self.tx_script.as_ref()
+        self.tx_args.tx_script()
+    }
+
+    /// Returns a reference to the note args.
+    pub fn note_args(&self) -> Option<&BTreeMap<NoteId, Word>> {
+        self.tx_args.note_args()
     }
```

Comment:
Similar comment as above: I wonder if we should replace these with a single `tx_args()` method.